docs/guides/user/search-query-guide.md

```diff
@@ -185,9 +184,9 @@ This query will find all the potential Remote Desktop event log entries in the g
 
 ### Advanced search
 
-Advanced search queries are in JSON format,  and let you use the full power of Elasticsearch. You can view your existing Query String query as an advanced Elasticsearch query by clicking "Advanced" button below the query entry field.
+Advanced search queries are in JSON format, and let you use the full power of OpenSearch. You can view your existing Query String query as an advanced OpenSearch query by clicking "Advanced" button below the query entry field.
 
-[Full Elasticsearch guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html)
+[Full query DSL guide](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html)
```

Comment:
We should also link here to https://opensearch.org/docs/latest/opensearch/query-dsl/index/

Reply:
fixed.